lib/zlib.js

```diff
@@ -899,6 +901,14 @@ function createProperty(ctor) {
   };
 }
 
+function crc32(data, value = 0) {
+  if (typeof data !== 'string' && !isArrayBufferView(data)) {
```

Comment:
Any reason not to allow `ArrayBuffer` or string coercion?

Reply:
Not really other than I am not super keen on taking care of something users can already wrap around in the first PR unless there is already an existing C++ utility that works well with all types of inputs though there isn’t any currently AFAICT. Follow-up welcomed.

Reply:
Although implicit string coercion sounds more like a footgun than a useful feature…users can coerce themselves that that might be less bug-prone?